### DISCUSSION TITLE
A list of all addresses in use

### DISCUSSION THREAD

**Discussion by shanse01** [Supergiovane/node-red-contrib-knx-ultimate#343] [Ideas]
Hello, 

and first of all thanks for the great work!

Resolving a problem where I moved a KNX device to another line and missing out on the correct routing (addresses in dummy devices) I could not find any possibility to have a list of all addresses used in active nodes.

I think this would be a great help in maintaining filter tables / dummy devices.

In case I missed a feature, please excuse my ignorance.

Best regards
Sven

**Comment by Supergiovane** [comment]
Hi shanse
Will start working on it yet.
I'll let you know, as soon as i release the new version. Should be out in about 1 hour.

**Comment by Supergiovane** [comment]
node-red-contrib-knx-ultimate@2.4.13 is out.
You can find the list here:

<img width="530" alt="Node-RED___Flow_2" src="https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/37666636/0bc4b497-919f-41f7-9057-e2f6208ab3e9">
